storybook/src/components/Button.js

```diff
@@ -0,0 +1,43 @@
+import React, {PureComponent} from 'react';
+import {CSSTransition} from 'react-transition-group';
+import "../css/Button.css";
+import classNames from 'classnames';
+
+export class Button extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            label: props.label,
+            round: props.round,
+            classNames: "custom-button ",
+            id: this.props.id,
+            disabled: this.props.disabled,
+        };
+        if(props.className) {
+            this.state.classNames = this.state.classNames + " " + props.className;
+        }
+
+        if(this.state.round) {
+            this.state.classNames = this.state.classNames + " custom-button__corner--round";
+        }
+        else {
+            this.state.classNames = this.state.classNames + " custom-button__corner--default";
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        if(prevProps.disabled !== this.props.disabled) {
+            this.setState({
```

Comment:
I don't think this is necessary if you just use the props directly; however, this is a good spot for animations on becoming enabled or becoming disabled.